src/mmw/js/src/core/layerPicker.js

```diff
@@ -127,21 +128,67 @@ var LayerPickerGroupView = Marionette.LayoutView.extend({
         layers: '#layerpicker-layers',
     },
 
+    modelEvents: {
+        'change': 'render'
+    },
+
     initialize: function(options) {
         this.leafletMap = options.leafletMap;
     },
 
     onRender: function() {
-        this.showChildView('layers', new LayerPickerLayerListView({
-          collection: this.model.get('layers'),
-          model: this.model,
-          leafletMap: this.leafletMap,
-      }));
+        if (this.model.get('name') === "Observations" && !this.model.get('layers')) {
+            this.fetchObservations();
+        } else {
+            this.showChildView('layers', new LayerPickerLayerListView({
+                collection: this.model.get('layers'),
+                model: this.model,
+                leafletMap: this.leafletMap,
+            }));
+        }
+    },
+
+    fetchObservations: function () {
+        var self = this,
+            pointSrcAPIUrl = '/api/modeling/point-source/';
+        this.model.set('polling', true);
+        $.when(self.model.get('layersDeferred'), $.ajax({ 'url': pointSrcAPIUrl, 'type': 'GET'}))
+            .done(function(observationLayers, pointSourceData) {
+                self.model.set('polling', false);
+                self.model.set('error', null);
```

Comment:
There's a shorthand for assigning multiple values in Backbone `model.set({ polling: false, error: null })`.